src/control/control.c

```diff
@@ -664,6 +670,22 @@ int dt_control_key_pressed_override(guint key, guint state)
   /* check if key accelerators are enabled*/
   if(darktable.control->key_accelerators_on != 1) return 0;
 
+  // dynamic accels
+  darktable.view_manager->current_view->dynamic_accel_current = dt_dynamic_accel_find_by_key(key, state);
+  if(darktable.view_manager->current_view->dynamic_accel_current)
+  {
+    gchar **vals = g_strsplit_set(darktable.view_manager->current_view->dynamic_accel_current->path, "/", -1);
+    gchar *txt = "";
+    if(vals[0] && vals[1] && vals[2] && vals[3])
+    {
+      txt = dt_util_dstrcat(NULL, "scroll to change <b>%s</b> of %s module", vals[3], vals[2]);
+    }
+    dt_control_hinter_message(darktable.control, txt);
+    g_free(txt);
```

Comment:
and you free it there.

It may explain the crash I have with this PR each time I exit the preference dialog.